src/common/marlin_server.hpp

```diff
@@ -25,6 +25,9 @@ void fsm_change(ClientFSM type, T phase, fsm::PhaseData data) {
     _fsm_change(type, fsm::BaseData(GetPhaseIndex(phase), data));
 }
 
+// true if user can stop waiting for heating/cooling by pressing a button
+extern bool can_stop_wait_for_heatup;
```

Comment:
don't extern variables
include header containing getter/setter function declaration

btw it seams to be unused, is it dead code?

Reply:
A lot of added code with no advantage.